website/ref/language.md

```diff
@@ -46,7 +46,7 @@ statements **executes** to produce side effects; an expression **evaluates** to
 some values. (The traditional terms for the two levels are "commands" and
 "words", but those terms are quite ambiguous.)
 
-# String
+# String data type
```

Comment:
I like it that you added "data type" to the section titles, but it would be even clearer if you make a new section called "data type" and move all the existing sections under it.